### ISSUE TITLE
xticks rotation being is ignored 

### ISSUE THREAD

**Issue by pranet** [mpld3/mpld3#449]
I am trying to rotate the labels on the x axis by using plt.xticks(positions, labels, rotation=90). While it is working fine using plt.show(), mpld3.show() is completely ignoring the rotation parameter and displaying text horizontally (0 rotation) instead.

Can someone suggest a workaround please?

**Comment by azayz** [comment]
Yes please I'm also struggling with this, the title doesn't appear aswell 
![Capture](https://user-images.githubusercontent.com/40893766/86252112-14291c00-bbb3-11ea-9aa5-53c0f2437869.PNG)

**Comment by cliffckerr** [comment]
Good question @azayz @pranet -- we'll take a look!

**Comment by vladh** [comment]
I'm working on this and I'll update you as soon as it's fixed! Sorry for the delay, trying to make time. :)

**Comment by vladh** [comment]
@azayz I'm currently testing this, is there any chance you could share your code, so I can confirm I'm actually fixing it for your use case? Somehow I'm experiencing even more bugs than you did.

Also, related issues: #369 #22 #221 #360 #364

**Comment by vladh** [comment]
I've looked at this in more detail, and have reached the conclusion that tick rotation seems to be a feature that was never implemented, and probably needs to be implemented in both mplexporter and `mpld3.js`, as I don't think the rotation information is currently passed from MPL to D3. The rotation information should be extracted in `mplexporter/mplexporter/utils.py` and added to the D3 graph in `src/core/axis.js`. I'll do my best to make time to implement this, which should go more quickly now that I've identified the issue, but right now tick rotation is not something that mpld3 supports. I'll try to add it soon! :)

**Comment by munna** [comment]
As per me, a simple and easy solution to rotate xAxis is CSS.

`css = """
.mpld3-xaxis text{
        transform: translate(18px, 30px) rotate(45deg);
    }
    """`

Apply this CSS to plugins connect
`mpld3.plugins.connect(fig, mpld3.plugins.PointHTMLTooltip(lines[0],labels=labels,voffset=10, hoffset=10,css=css))        `

It works for me like charm.

**Comment by asilvino** [comment]
Do we have any update on this matter?

**Comment by vladh** [comment]
I won't have the time to fix this myself anytime soon, but I would gladly review a PR and I encourage those who this affects to send one!

**Comment by JosePereiraUA** [comment]
Hi.
I too would like an update on this issue.
+1 if someone could take a look.
Cheers!
